### PR TITLE
Update tutorial section 01

### PR DESCRIPTION
**Pull request by murtza**
Update tutorial section 01

### CODE REVIEW

File: Sources/LiveViewNative/LiveViewNative.docc/Resources/01-01-03-config01.exs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  metadata: [:request_id]
+
+# Use Jason for JSON parsing in Phoenix
+config :phoenix, :json_library, Jason
+
+config :live_view_native, :platforms, [LiveViewNativeSwiftUi.Platform]
+
+config :live_view_native, LiveViewNativeSwiftUi.Platform,
+  app_name: "LVNTutorial",
+  custom_modifiers: []
```
Comment (shadowfacts) [inline Sources/LiveViewNative/LiveViewNative.docc/Resources/01-01-03-config01.exs:54]: Can you move these closer to the top of the file? The rendered tutorial doesn't do a great job indicating when there are changes below the fold, which can confuse people.

**Review by shadowfacts — CHANGES_REQUESTED** [review]
The new tutorial resource files should be renamed to match the convention (`<tutorial number>-<section number>-<file number>`), and the ones following the new files should have their numbers adjusted, but other than that, this looks good!

**Comment by murtza** [comment]
> The new tutorial resource files should be renamed to match the convention (`<tutorial number>-<section number>-<file number>`), and the ones following the new files should have their numbers adjusted, but other than that, this looks good!

What do you think about following convention `<tutorial number>-<section number>-<file number>-<file name><change number>`. This way we can show originally generated file and the changes will be highlighted.

**Comment by shadowfacts** [comment]
Right now, the initial versions of files are already included (like `02-01-02-application.ex` is the initial version, and `02-01-03-application.ex` is the modified version). I'm fine with either convention, so long as we're consistent throughout the resource files.

**Comment by murtza** [comment]
Ok, let's follow the existing convention `<tutorial number>-<section number>-<file number>`.

File: Sources/LiveViewNative/LiveViewNative.docc/Tutorials/01 Initial List.tutorial [hunk 2]
```diff
@@ -89,13 +116,22 @@
...
+               Edit the `app.html.heex` file in the created project.
+               
+               @Comment {
+                   skip = true
+               }
+               @Code(name: "lib/lvn_tutorial_web/templates/layout/app.html.heex", file: 01-01-08-app-template.html.heex)
+           }
+
             @Step {
                 Lastly, we need to tweak the `live.html.heex` layout to remove the extraneous HTML that the native frontend doesn't need.
```
Comment (shadowfacts) [inline Sources/LiveViewNative/LiveViewNative.docc/Tutorials/01 Initial List.tutorial:129]:
```suggestion
                Lastly, we need to tweak the `app.html.heex` layout to remove the extraneous HTML that the native frontend doesn't need.
```

**Review by shadowfacts — APPROVED** [review]

**Pull request merged**